awx/ui/client/src/shared/branding/colors.default.less

```diff
@@ -1,7 +1,7 @@
 // default base colors
-@default-interface-txt: #707070;
+@default-interface-txt: #606060;
```

Comment:
Interface text throughout was slightly too light in various places, this minor adjustment makes us pass.